.github/workflows/prod_deploy.yaml

```diff
@@ -7,18 +7,27 @@ jobs:
   invoke-prod-deploy:
     runs-on: ubuntu-latest
     steps:
-      - name: deploy
+      - name: deploy_gcp
         run: |
           DEPLOY_BATCH_URL=$(curl --fail --silent --show-error -X POST \
               -H "Authorization: Bearer ${{ secrets.CI_TOKEN }}" \
               -H "Content-Type:application/json" \
               -d '{"steps": ["deploy_auth", "deploy_batch", "deploy_ci", "deploy_hailgenetics_image", "deploy_memory", "deploy_wheel", "upload_query_jar"], "sha": "${{ github.sha }}"}' \
               https://ci.hail.populationgenomics.org.au/api/v1alpha/prod_deploy)
-          echo DEPLOY_BATCH_URL=$DEPLOY_BATCH_URL >> $GITHUB_ENV
+          echo DEPLOY_BATCH_URL_GCP=$DEPLOY_BATCH_URL >> $GITHUB_ENV
+
+      - name: deploy_azure
+        run: |
+          DEPLOY_BATCH_URL=$(curl --fail --silent --show-error -X POST \
+              -H "Authorization: Bearer ${{ secrets.CI_TOKEN_AZURE }}" \
+              -H "Content-Type:application/json" \
+              -d '{"steps": ["deploy_auth", "deploy_batch", "deploy_ci"], "sha": "${{ github.sha }}"}' \
+              https://ci.azhail.popgen.rocks/api/v1alpha/prod_deploy)
+          echo DEPLOY_BATCH_URL_AZURE=$DEPLOY_BATCH_URL >> $GITHUB_ENV
 
       - name: post to Slack
         run: |
           curl --fail --silent --show-error -X POST \
               -H "Content-type: application/json" \
-              -d "{\"text\": \"Deploying Hail Batch: $DEPLOY_BATCH_URL\"}" \
+              -d "{\"text\": \"Deploying Hail Batch: $DEPLOY_BATCH_URL_GCP $DEPLOY_BATCH_URL_AZURE\"}" \
```

Comment:
Can we put the two URLs on separate lines and make really clear what refers to GCP vs Azure?

Reply:
Sure!